static/public.html

```diff
@@ -0,0 +1,289 @@
+</svg>`;
+      const defaultContent = `# Calque - reactive calculator
+        # Documentation: https://github.com/grimalschi/calque`;
+      const firstContent = `# Calque - reactive calculator
+# Expressions
+2 + 2 * 2
+sqrt(3^2 + 4^2 + 5^2 + 6^2 + 7^2 + 8^2)
+2 inch to cm
+cos(45 deg)
+# Variables
+a = 25
+b = a * 5
+postal code = 1122
+# Summing lists
+animals:
+  cats = 2
+  dogs = 3
+plants:
+  trees = 20
+  vegetables:
+    potatoes = 10
+    carrots = 10
+# Functions
+pow2(x) = x ^ 2
+g(x) = x * 2
+pow2(g(4))
+# Last result
+2 * 2
+last + 1
+# Keys
+# Duplicate line or selection: Ctrl+D
+# Change selected number: Up/Down
+# Change selected number 10x: Shift+Up/Down
+# Change indent: Tab/Shift+Tab
+# Calque on GitHub:
+# https://github.com/grimalschi/calque
+# Using Math.js:
+# https://github.com/grimalschi/mathjs`;
+
+      // effects
+      // misc fx
+
+      const renderIcons = (dispatch, options) => {
+        requestAnimationFrame(() => {
+          feather.replace();
+        });
+      };
+      const attachCalq = (dispatch, options) => {
+        requestAnimationFrame(() => {
+          let input = document.getElementById("input");
+          let output = document.getElementById("output");
+          window.calque = new Calque(input, output);
+          input.focus();
+        });
+      };
+
+      const loadNbRecords = async (dispatch, options) => {
+        var parts = window.location.pathname.split("/");
+        var lastSegment = parts.pop() || parts.pop();
+        const req = await fetch(`/public/raw/${lastSegment}`, {
```

Comment:
Curious on the implementation here?

What does `/public/raw/:segment` do?

Reply:
it returns the public note record
`public/:segment` returns the page, and `/public/raw/:segment` returns the note in a list like this 
[{name: "notebook 1", content: "jdhdh", active: true, public: true}]